### PR TITLE
Enable jsf23 general and repeat tests for ee10

### PR DESCRIPTION
**Pull request by pnicolucci**
for #22368

### CODE REVIEW

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
#libby #build

File: dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23GeneralTests.java [hunk 1]
```diff
@@ -626,24 +633,27 @@ public void testSpecIssue1113() throws Exception {
...
-                Log.info(c, name.getMethodName(), "FailingHttpStatusCodeException response: " + response);
-                Log.info(c, name.getMethodName(), "FailingHttpStatusCodeException statusCode: " + statusCode);
-
-                /*
-                 * com.ibm.ws.jsp.translator.JspTranslationException:
-                 * JSPG0123E: Unable to locate tag attribute info for tag attribute onselect.<br>
-                 */
-                assertTrue("The JSP was rendered successfully and it should not have been.", response.contains("JSPG0123E") && statusCode == 500);
+            // Faces 4.0 does not support Pages any longer!
+            if (!JakartaEE10Action.isActive()) {
```
Comment (volosied) [inline dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23GeneralTests.java:637]: I don't understand the change here as this test is now skipped for EE10?

**Review by volosied — APPROVED** [review]

File: dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23GeneralTests.java (hunk 1 above)
Comment (pnicolucci) [reply dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23GeneralTests.java:637]: different tests, git made that a bit confusing, check line numbers.

**Comment by pnicolucci** [comment]
#libby

**Pull request merged**